aiven/client/client.py

```diff
@@ -2134,3 +2135,410 @@ def delete_oauth2_client_secret(self, account_id: str, client_id: str, secret_id
             self.delete,
             self.build_path("account", account_id, "oauth_client", client_id, "secret", secret_id),
         )
+
+    def flink_list_applications(
+        self,
+        *,
+        project: str,
+        service: str,
+    ) -> Mapping:
+        return self.verify(
+            self.get,
+            self.build_path(
+                "project",
+                project,
+                "service",
+                service,
+                "flink",
+                "application",
+            ),
+        )
+
+    def flink_create_application(
+        self,
+        *,
+        project: str,
+        service: str,
+        application_properties: Mapping[str, str],
```

Comment:
The type is not `Mapping[str, str]`. If the key is `application_version`, the value is an embedded object.

https://api.aiven.io/doc/#tag/Service:_Flink/operation/ServiceFlinkCreateApplication

Reply:
Done, I changed to `str, Any`.